fastify.js

```diff
@@ -268,8 +268,8 @@ function build (options) {
             reject(err)
           } else {
             server.removeListener('error', errEventHandler)
-            logServerAddress(server.address(), options.https)
-            resolve()
+            var address = logServerAddress(server.address(), options.https) || null
```

Comment:
When will this ever be `null`? Can you add a test for such a case?

Reply:
Sorry for the typo it wouldn't be null. `logServerAddress()` always returning address that. I added test case for that.